### ISSUE TITLE
guided hybrid attention model

### ISSUE THREAD

**Issue by Hongzl1996** [JianshuZhang/TAP#1]
Hi, jianshu,
In v2, I can't found code about 'guided hybrid attention model'? Is this version consistent with the model described in your paper?
![image](https://user-images.githubusercontent.com/25878132/47647806-17d0f800-dbb3-11e8-98f6-fc9d737fbbd3.png)

**Comment by JianshuZhang** [comment]
Sorry, it’s my mistake, v2 only contains the attention guider, I will update the temporal attention.

**Comment by JianshuZhang** [comment]
@Hongzl1996 Hi, you can find the hybrid attention code now. To be quick, I just update my experiment code. I will soon unify the codes.

**Comment by Hongzl1996** [comment]
Thank you for your quick answer. By the way, How cool the attention idea in TAP is !

**Comment by Hongzl1996** [comment]
@JianshuZhang  Hi, I found the input sequence is 9-dimension in this version, while the TAP paper shows 8-dimension?
I am trying to reimplement TAP by pytorch recently, so I am wondering some details. What's more,  Is there an introduction about the data you  supply in github(Is it all from CROHME 2014 or something)?
Looking forward to your reply. Thanks.

**Comment by JianshuZhang** [comment]
Yes, but the difference of 8-dimension and 9-dimension features can be ignored, the extra one dimensional feature has no useful meaning, so I didn’t write it in the paper, and there is no effect on performance. So, you can use the 9-dimension features on GitHub or use the 8-dimension features.

The details of the data and normalization method has been introduced in the TAP paper, or you can refer to paper 'Drawing and Recognizing Chinese Characters with Recurrent Neural Network'
